apps/api/src/app/events/events.controller.ts

```diff
@@ -39,4 +31,31 @@ export class EventsController {
       })
     );
   }
+
+  private mapSubscribers(body: TriggerEventDto) {
+    const subscribers = Array.isArray(body.to) ? body.to : [body.to];
+    const mappedSubscribers: ISubscribersDefine[] = subscribers.map((subscriber) => {
+      EventsController.validateSubscriberIdProperty(subscriber);
+
+      if (typeof subscriber === 'string') {
+        return {
+          subscriberId: subscriber,
+        };
+      } else {
+        return subscriber;
+      }
+    });
+
+    return mappedSubscribers;
+  }
+
+  private static validateSubscriberIdProperty(subscriber) {
```

Comment:
Can we provide optional types for providers here?

Reply:
Done 👍